master/master.cfg

```diff
@@ -238,8 +238,12 @@ for branch_num, (git_url, branchname, git_branch) in enumerate(git_branches):
             tags.append(tier)
 
         # Only 3.11+ for WebAssembly builds
-        if "wasm" in tags and branchname in {"3.9", "3.10"}:
-            continue
+        if "wasm" in tags:
+            if branchname in {"3.9", "3.10"}:
+                continue
+            # Pydebug support is 3.13+
+            elif "nondebug" not in tags and branchname in {"3.11", "3.12"}:
+                continue
```

Comment:
@zware: Unrelated to this PR, it would be nice to have a way to describe these branches filters on builders, rather than here.

Reply:
I talked w/ Zach on Discord about this. I think we both agreed that such a feature would be nice.